include/seqan/basic/allocator_interface.h

```diff
@@ -204,6 +204,7 @@ allocate(T const &,
     if (data)
         SEQAN_PROADD(SEQAN_PROMEMORY, count * sizeof(TValue));
 #endif
+	memset((void*)data, 0, count * sizeof(TValue));
```

Comment:
I did some test commits to see the GCC and Clang results, test_sequence fails on Clang if I do 	memset((void*)data, 0xFFFFFFFF, count * sizeof(TValue));
So the must be a bug somewhere in seqan (seqan relying on zero initialized memory). I dont know how much performance penalty this additional memset is, maybe somebody who know this low level code should look into it???